package.json

```diff
@@ -47,29 +47,32 @@
         "mustache": "4.2.0",
         "prettier": "2.5.1",
         "semver": "7.3.5",
-        "sfmc-sdk": "0.3.0",
-        "simple-git": "2.48.0",
+        "sfmc-sdk": "0.4.0",
+        "simple-git": "3.2.6",
         "sql-formatter-plus": "1.3.6",
         "toposort": "2.0.2",
         "update-notifier-git": "5.0.3",
-        "winston": "3.4.0",
+        "winston": "3.6.0",
         "yargs": "17.3.1"
     },
     "devDependencies": {
-        "chai": "4.3.4",
-        "eslint": "8.6.0",
-        "eslint-config-prettier": "8.3.0",
+        "chai": "4.3.6",
+        "eslint": "8.9.0",
+        "eslint-config-prettier": "8.4.0",
         "eslint-config-ssjs": "1.1.11",
         "eslint-plugin-mocha": "10.0.3",
         "eslint-plugin-prettier": "4.0.0",
         "husky": "7.0.4",
-        "jsdoc-to-markdown": "7.1.0",
-        "lint-staged": "12.1.7",
-        "mocha": "9.1.3",
+        "jsdoc-to-markdown": "7.1.1",
+        "lint-staged": "12.3.4",
+        "mocha": "9.2.1",
```

Comment:
can we please stop hiding lots of dependency upgrades in pull requests? there is not even a single commit dedicated to this. if something goes wrong because of the dep upgrade we'll have a hard time even noticing that as the root cause

- [ ] please revert